extension/llm/export/partitioner_lib.py

```diff
@@ -138,9 +139,16 @@ def get_qnn_partitioner(
     if pt2e_quantize is not None:
         use_fp16 = False
 
+    soc_chip_table = {
+        "SM8650": QcomChipset.SM8650,
+        "SM8550": QcomChipset.SM8550,
+        "SM8475": QcomChipset.SM8475,
+        "SM8450": QcomChipset.SM8450,
+    }
```

Comment:
I think we need have this table somewhere in the qualcomm backend

Reply:
let me change it to
```
    qnn_partitioner = QnnPartitioner(
        generate_qnn_executorch_compiler_spec(
            soc_model=getattr(QcomChipset, soc_model),
            backend_options=backend_options,
            debug=False,
            saver=False,
            shared_buffer=shared_buffer,
            profile=False,
        ),
        skip_node_id_set,
        skip_node_op_set,
    )
```

Reply:
oh is it possible to have a function inside qualcomm like
```
def get_soc_table():
      return soc_chip_table = {
        "SM8650": QcomChipset.SM8650,
        "SM8550": QcomChipset.SM8550,
        "SM8475": QcomChipset.SM8475,
        "SM8450": QcomChipset.SM8450,
    }
```
and call it here?

Reply:
that's actually what I meatn..and the previous version seems good actually

Reply:
Thanks for your advice.
But I think we don't want to add this table for maintenance convenience.
Because if we would like to add new soc, we will need to maintain extra table.